setuptools_rust/build.py

```diff
@@ -150,9 +137,6 @@ def build_extension(
 
         rustflags = []
 
-        if linker is not None:
-            rustflags.extend(["-C", "linker=" + linker])
```

Comment:
As per discussion here I am now 100% in agreement that this was wrong and it is correct to remove linker handling from `setuptools-rust`.